lib/twine/cli.rb

```diff
@@ -45,6 +45,14 @@ module CLI
           files are UTF-16 without BOM, you need to specify if it's UTF-16LE or UTF16-BE.
         DESC
       },
+      escape_all_tags: {
+        switch: ['--[no-]escape-all-tags'],
```

Comment:
Just thinking out loud -- would it make sense to have a flag that lets you escape all, none, or the current (default)?

Reply:
Mmmhh...I don't think so. As far as I know there's no method on Android to load a string that preserves un-escaped non-styling tags.